### PR TITLE
Check for uppercase SHOW TABLE STATUS result

### PR DESCRIPTION
**Pull request by Jason-Morcos**
What does this implement/fix? Explain your changes.
---------------------------------------------------
Allows the database list view to handle if SHOW TBLE STATUS returns uppercase column names.
This is the smallest fix I could come up with. There may be a far better way of handling this.

Does this close any currently open issues?
------------------------------------------
Fixes (possibly) #177

### CODE REVIEW

File: Source/SPTablesList.m [hunk 1]
```diff
@@ -219,13 +219,19 @@ - (IBAction)updateTables:(nullable id)sender
 				// Due to encoding problems it can be the case that [resultRow objectAtIndex:0]
 				// return NSNull, thus catch that case for safety reasons
 				id tableName = [eachRow objectForKey:@"Name"];
```
Comment (Kaspik) [inline Source/SPTablesList.m:221]: `id tableName = [eachRow objectForKey:@"Name"] ?? [eachRow objectForKey:@"NAME"];` :)

File: Source/SPTablesList.m [hunk 2]
```diff
@@ -219,13 +219,19 @@ - (IBAction)updateTables:(nullable id)sender
...
+				if ([tableName isNSNull]) {
+					tableName = [eachRow objectForKey:@"NAME"];
+				}
 				if ([tableName isNSNull]) {
 					tableName = @"...";
 				}
 				[tables addObject:tableName];
 				
 				// comments is usefull
 				id tableComment = [eachRow objectForKey:@"Comment"];
```
Comment (Kaspik) [inline Source/SPTablesList.m:231]: ^^

Comment (Jason-Morcos) [reply Source/SPTablesList.m]: Better? haha

**Comment by Jason-Morcos** [comment]
> Think we need nil checks per [#177 (comment)](https://github.com/Sequel-Ace/Sequel-Ace/issues/177#issuecomment-655303689).
> 
> I tested your code on the alibaba db and it still fails.
> 
> ```objc
> id tableName = [eachRow objectForKey:@"Name"] ?: [eachRow objectForKey:@"NAME"];
> // dictionaries return nil, not a null object if there is no value is associated with the Key.
> if ([tableName isNSNull] || tableName == nil ) {
>      tableName = @"...";
> }
> [tables addObject:tableName];
> 				
> // comments is useful
> id tableComment = [eachRow objectForKey:@"Comment"] ?: [eachRow objectForKey:@"COMMENT"];
> if ([tableComment isNSNull] || tableComment == nil ) {
> 	tableComment = @"";
> }
> [tableComments setValue:tableComment forKey:tableName];
> ```
> 
> Probably best to put the `tableComment == nil`first in the if statement actually.

Made this change!

**Pull request merged**